scripts/noaa/gpcc_spi/manifest.json

```diff
@@ -0,0 +1,112 @@
+{
+    "import_specifications": [
+        {
+            "import_name": "NOAA_GPCC_StandardardizedPrecipitationIndex",
+            "curator_emails": [
+                "support@datacommons.org"
+            ],
+            "provenance_url": "https://www.drought.gov/data-maps-tools/global-precipitation-climatology-centre-gpcc-standardized-precipitation-index-spi",
+            "provenance_description": "SPI for 1 degree grids calculated from monthly precipitation data using the Pearson distribution.",
+            "scripts": [
+                "download.py",
+                "preprocess_gpcc_spi.py"
+            ],
```

Comment:
Since NOAA_GPCC_StandardardizedPrecipitationIndex_AggPlace below is already invoking these scripts,
we can skip this and have the manifest for NOAA_GPCC_StandardardizedPrecipitationIndex refer to the latest version for NOAA_GPCC_StandardardizedPrecipitationIndex_AggPlace.

Reply:
Changed the manifest.json to one import which invokes all the files.